int8 id                 # Id of the servo on the Pololu board
string name             # Name of the servo (specified in the yaml file)
float64 position        # Current position of the servo in radians
float64 position_deg    # Current position of the servo in degrees
